git_toprepo.py

```diff
@@ -1510,13 +1521,24 @@ def get_commits_to_fetch() -> Set[CommitHash]:
                 commit_hash_str = commit_hash.decode("utf-8")
                 print(f"\trev-{commit_hash_str} = {url}")
             if len(missing_commits) > max_missing_commits_to_print:
-                print(f"Totally {len(missing_commits)} unexpected missing commits.")
-            print("ERROR: The referenced commits above could not be found")
+                print(
+                    f"{len(missing_commits)} unexpected missing commits.",
```

Comment:
I'd clarify the message here with something like:
```suggestion
                    f"and {len(missing_commits) - max_missing_commits_to_print} more unexpected missing commits.",
```